tests/unit_functional

```diff
@@ -12,20 +12,21 @@ set +a
 
```

Comment:
I think the changes in this file have been included in this pull request by mistake?